release_notes/ocp_3_6_release_notes.adoc

```diff
@@ -0,0 +1,2471 @@
+you would like to make sure another component of that application lands in the
+same zone.
+
+Another use case is if you have two application components that, due to security
+reasons, cannot be on the same physical box. However, you do not want to lock
+them into labels on nodes. You want them to land anywhere, but still honor
+anti-affinity.
+
+Many of the same high-level concepts mentioned in the node affinity and
+anti-affinity hold true here. For pods, you declare a
+link:https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#inter-pod-affinity-and-anti-affinity-beta-feature[`topologyKey`],
+which will be used as the boundary object for the placement logic.
+
+----
+affinity:
+    podAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchExpressions:
+          - key: service
+            operator: In
+            values: [“S1”]
+        topologyKey: failure-domain.beta.kubernetes.io/zone
+
+
+affinity:
+    podAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchExpressions:
+          - key: service
+            operator: In
+            values: [“S1”]
+        topologyKey: kubernetes.io/hostname
+----
+
+See
+xref:../admin_guide/scheduling/pod_affinity.adoc#admin-guide-sched-pod-affinity[Advanced
+Scheduling and Pod Affinity and Anti-affinity] for more information.
+
+[[ocp-36-taints-and-tolerations]]
+==== Taints and Tolerations
+
+xref:../admin_guide/scheduling/taints_tolerations.adoc#admin-guide-taints[Taints
+and tolerations] allow the *node* to control which *pods* should (or should
+not) be scheduled on them.
+
+A _taint_ allows a node to refuse pod to be scheduled unless that pod has a
+matching _toleration_.
+
+You apply taints to a node through the node specification (`NodeSpec`) and apply
+tolerations to a pod through the pod specification (`PodSpec`). A taint on a
+node instructs the node to repel all pods that do not tolerate the taint.
+
+Taints and tolerations consist of a key, value, and effect. An operator allows
+you to leave one of these parameters empty.
+
+In {product-title} 3.6, daemon pods do respect taints and tolerations, but they
+are created with `NoExecute` tolerations for the
+`node.alpha.kubernetes.io/notReady` and `node.alpha.kubernetes.io/unreachable`
+taints with no `tolerationSeconds`. This ensures that when the
+`TaintBasedEvictions` alpha feature is enabled, they will not be evicted when
+there are node problems such as a network partition. (When the
+`TaintBasedEvictions` feature is not enabled, they are also not evicted in these
+scenarios, but due to hard-coded behavior of the `NodeController` rather than
+due to tolerations).
+
+Set the taint from the command line:
+
+----
+$ oc taint nodes node1 key=value:NoSchedule
+----
+
+Set toleration in the `PodSpec`:
+
+----
+tolerations:
+- key: "key"
+  operator: "Equal"
+  value: "value"
+  effect: "NoSchedule"
+----
```

Comment:
> @bparees @deads2k Just this section, or the Taints and Tolerations docs too? Can you please let me know what needs fixed in both sections?

The fields are there in 3.6.  I must just have remembered the various bugs in the conversion logic.